cmd/vcluster/context/context.go

```diff
@@ -62,6 +62,8 @@ type VirtualClusterOptions struct {
 
 	DefaultImageRegistry string `json:"defaultImageRegistry"`
 
+	PodSecurityStandard string `json: enforcePodSecurityStandard`
```

Comment:
Wrong format, you need to do:

PodSecurityStandard string `json:"podSecurityStandard"`

Reply:
ahh, I changed the type and then forgot to change its format.